app/models/exporters/tableau.rb

```diff
@@ -4,5 +4,185 @@ module Exporters::Tableau
 
   module_function
 
+    def null
+      lit 'NULL'
+    end
+    private_class_method :null
+
+    def entry_exit(start_date: 3.years.ago, end_date: DateTime.current, coc_code: nil)
+      model = she_t.engine
+
+      spec = {
+        data_source:                      c_t[:data_source_id],
```

Comment:
I think, maybe, this should be `she_t[:data_source_id]`

Reply:
Well, this is used for joining income/benefits to clients, so I think they will all be the same, and in any case we really want this to be the same as whoever we've joined in as a client. I think it will all be okay.